test/e2e/framework/ingress_utils.go

```diff
@@ -1057,10 +1058,39 @@ func (j *IngressTestJig) CreateIngress(manifestPath, ns string, ingAnnotations m
 		j.Ingress.Annotations[k] = v
 	}
 	Logf(fmt.Sprintf("creating" + j.Ingress.Name + " ingress"))
-	j.Ingress, err = j.Client.ExtensionsV1beta1().Ingresses(ns).Create(j.Ingress)
+	j.Ingress, err = j.RunCreate(j.Ingress)
 	ExpectNoError(err)
 }
 
+// RunCreate runs the required command to create the given ingress.
+func (j *IngressTestJig) RunCreate(ing *extensions.Ingress) (*extensions.Ingress, error) {
```

Comment:
Could we simply name this `CreateIngress`?

Reply:
There already exists a `CreateIngress` method . This is specific to running the Create command. CreateIngress calls RunCreate once it has computed the ingress to create.

Reply:
Oops, you are right, I scanned too quick. Maybe `RunCreateIngress`? Just wanted to indicate this is ingress specific :)

Reply:
Its already a member function of IngressTestJig so RunCreate should be enough?
For creating other resources like services, we can name it explicitly RunCreateService.

I think RunCreate is good, but if you still feel strongly am happy to change it to RunCreateIngress.

Reply:
Yeah no worries, `RunCreate` should be enough